index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Apple</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="./assets/stylesheets/index.css" />
+  </head>
+
+  <body>
+    <main>
+      <div class="hero-section">
+        <nav>
+          <ul>
+            <li class="nav-item">
+              <a href="#"><i class="fa fa-apple logo"></i> </a>
+            </li>
+            <li class="nav-item"><a href="#">Store</a></li>
+            <li class="nav-item"><a href="#">Mac</a></li>
+            <li class="nav-item"><a href="#">iPod</a></li>
+            <li class="nav-item"><a href="#">iPad</a></li>
+            <li class="nav-item"><a href="#">iTunes</a></li>
+            <li class="nav-item"><a href="#">Support</a></li>
+            <li class="nav-item">
+              <div class="searchbar">
+                
+                <input type="text"/>
+              </div>
+            </li>
+          </ul>
+        </nav>
+        <div class="hero-text">
+          <h1>
+            What will your verse be?
+          </h1>
+          <h2>
+            Everyone has something to share. See how people are using iPad in
+            their unique way to add to the world's story.
+          </h2>
+          <p>
+            Watch the "Your Verse" film and explore the stories
+            <img src="./assets/img/play.png" alt="play icon" />
+          </p>
+        </div>
+      </div>
+      <div class="after-hero">
+        <ul>
+          <li>
+            <a href="#"><img src="./assets/img/apple5.jpg" alt="apple 5" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./assets/img/iphone5c.jpg" alt="apple 5" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./assets/img/apple_30_years.jpg" alt="apple 5"/></a>
+          </li>
+          <li>
+            <a href="#"><img src="./assets/img/apple_supplier_responsibility.jpg" alt="apple 5" /></a>
```

Comment:
- [ ] In addition, since this is a project based on backgrounds and gradients kindly make these four images here backgrounds instead.